CreateSnapshot/src/test/java/org/opensearch/migrations/TestCreateSnapshot.java

```diff
@@ -0,0 +1,115 @@
+package org.opensearch.migrations;
+
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
this is a bit risky since you're (I think) depending on a transitive dependency.  
For a test case, it's reasonable to take on extra dependencies that you normally wouldn't want to take on.
That said, the lack of a Pair<> type is well-known for Java devs.  Normally, we just do Map.Entry<> for the type and instantiate `new AbstractMap.SimpleEntry<>(V1, V2)`.  For test code, I'm fine w/ another dependency on this or vavr.io or whatever, it will probably eventually be refactored away to keep dependency graphs smaller.